src/desktop/open_uri.rs

```diff
@@ -233,8 +236,12 @@ impl<'a> OpenURIProxy<'a> {
         file: &impl AsRawFd,
         writeable: bool,
         ask: bool,
+        activation_token: Option<ActivationToken>,
     ) -> Result<(), Error> {
-        let options = OpenFileOptions::default().ask(ask).writeable(writeable);
+        let mut options = OpenFileOptions::default().ask(ask).writeable(writeable);
+        if let Some(token) = activation_token {
```

Comment:
Make the option take ActivationToken instead of a string & impelement Serialize on it

Reply:
Are there other portals that take this option and we don't handle currently? needs to be checked

Reply:
Seems not, there is DynamicLauncher portal if you want to do it. It needs a new struct to define the options for it though as it doesn't have one for now

Reply:
Do we really need a struct? I used a simple hashmap here as it was already the case.

Reply:
The struct makes it super easy to enforce the type system instead of a random map somewhere